coffee-shop/src/pages/Details/components/card/Card.jsx

```diff
@@ -0,0 +1,11 @@
+import React from "react"
+import styles from "../card/Card.module.css"
+const Card = ({imageName}) => {
```

Comment:
I think Card is not the meaningful name for this component, I suggest something like `DetailsHeroImage`, The component folder should be the same as the component name even in letter cases

Reply:
Changes are done